### DISCUSSION TITLE
Fog of War/Dark Chess Variant

### DISCUSSION THREAD

**Discussion by sgaseretto** [niklasf/python-chess#867] [Q&A]
I just found about this library and I saw that it has some variants implemented, but could not find the Fog of War aka Dark Chess variant. Has anyone implemented that with this library?

**Comment by pavolkacej** [comment]
Hi @sgaseretto, I believe it is possible using this library. As far as I understand FoW, you can work with regular chess board from here, but it is just up to you which pieces you will be showing. This logic will be in your UI, and does not sound very complicated. I am not sure which UI are you using, but you may try Pygame to create chessboard and show pieces. Good luck!
